incorporate Adrian's comments (#58)
netmod-wg/schedule-yang: draft-ietf-netmod-schedule-yang.md

   *  "basic-recurrence"
   *  "icalendar-recurrence"

>    Refer to {{sec-aug}} and {{features}} for the use of these features.

Author: Unsure this is needed. It seems that just say {{sec-aug}} would be sufficient.

>    Refer to {{sec-aug}} and {{features}} for the use of these features.

Author: Note the Compilation error when using {{>sec-aug}}: 
Error: IDREF attribute target references an unknown ID "_sec-aug", at None

>    Refer to {{sec-aug}} and {{features}} for the use of these features.

Reviewer: Please check now. if we don't make this change this will be rendered as "Sections Section XX and Section xx". With this change we fix this. Thanks.

>    Refer to {{sec-aug}} and {{features}} for the use of these features.

Author: It works now. Thanks.

>    Refer to Sections {{sec-aug}} and {{features}} for the use of these features.

Reviewer: ```suggestion
   Refer to Sections {{>sec-aug}} and {{>features}} for the use of these features.
```

### The "recurrence-basic" Grouping {#sec-rec}

  The "recurrence-basic" grouping ({{rec-grp-tree}}) specifies a simple recurrence rule which starts immediately and repeats forever.

~~~~
{::include ./yang/tree/rec-grp.txt}
~~~~
{: #rec-grp-tree title="recurrence Grouping Tree Structure"}

>   The frequency parameter ("frequency") which is mandatory, identifies the type of recurrence rule. For example,

Reviewer: ```suggestion
  The frequency parameter ("frequency"), which is mandatory, identifies the type of recurrence rule. For example,
```

   The "start-time-utc" indicates the start time in UTC format.

   The "duration" parameter specifies, in units of seconds, the time period of the first occurrence. Unless specified otherwise (e.g., described in the "description" statement), the "duration" also applies to subsequent recurrence instances. When unspecified, each occurrence is considered as
   immediate completion or hard to compute an exact duration.

Note that the "interval" and "duration" cover two distinct properties of a schedule event.
The interval specifies when a schedule will occur, combined with the frequency parameter; while the duration indicates how long

> an occurrence will last. This document allows the interval between occurrences is shorter than the duration of each occurrence, e.g., a recurring event is scheduled to start every day for a duration of 2 days.

Reviewer: ```suggestion
an occurrence will last. This document allows the interval between occurrences to be shorter than the duration of each occurrence (e.g., a recurring event is scheduled to start every day for a duration of 2 days).
```

   *  The combination of the day, month, and year represented for date and time
      values MUST be valid. See {{Section 5.7 of ?RFC3339}} for the maxinum day
      number based on the month and year.
   *  The second for date and time values MUST have the value "60" at the end of months in which a leap
      second occurs.
   *  Schedules received with a starting time in the past with respect to

>       current time SHOULD be ignored. Some implementation MAY omit the past occurrences and

Reviewer: ```suggestion
      current time SHOULD be ignored. When a local policy is provided, an implementation MAY omit the past occurrences and
```

   such, there are no additional security issues related to the "ietf-
   schedule" module that need to be considered.

   Modules that use the groupings that are defined in this document

>    should identify the corresponding security considerations. For

Reviewer: ```suggestion
   should identify the corresponding security considerations.
```

>    example, reusing some of these groupings will expose privacy-related

Reviewer: ```suggestion
```

>    information (e.g., 'node-example').

Reviewer: ```suggestion
```

Author: security considerations used the template from rfc8407bis. Should this sentence also be removed from rfc8407bis?
PS. it is not really clear to me the example of node-example...

Reviewer: We need to keep "Modules that use the groupings that are defined in this document
   should identify the corresponding security considerations."

For the next one, we can remove it if we don't tag any node that we think should be carefully covered.

Author: Adrian left a comment with "I believe that section 7 is missing clear instructions to future module that make use of this module."  Please review the proposed the new text (preview at https://qiufangma.github.io/schedule-yang/draft-ietf-netmod-schedule-yang.html#section-7) in security considerations.
